docs/data-fetching.md

```diff
@@ -103,22 +103,22 @@ for a field you simply override this default resolver.
 **graphql-php** provides following default field resolver:
 ```php
 <?php
-function defaultFieldResolver($source, $args, $context, \GraphQL\Type\Definition\ResolveInfo $info)
+function defaultFieldResolver($rootValue, $args, $context, \GraphQL\Type\Definition\ResolveInfo $info)
 {
     $fieldName = $info->fieldName;
-    $property = null;
+    $property  = null;
```

Comment:
intentional?

Reply:
I think this is the autoformatting from the code style. I personally would not align stuff.

Reply:
Ops; sorry then